src/prpy/ik_ranking.py

```diff
@@ -53,13 +53,26 @@ def JointLimitAvoidance(robot, ik_solutions):
 
 
 class NominalConfiguration(object):
-    def __init__(self, q_nominal):
+    def __init__(self, q_nominal, max_deviation=2*numpy.pi):
```

Comment:
This default actually helps quite a lot on multirotation or continuous rotation robots.

The only special case where you might need explicit multirotation is where the intermediate trajectory needs to wrap a full rotation to avoid an intermediate obstacle that partitions the configuration space (like a corkscrew maze), which is uncommon unless a pathological environment is constructed.

On the other hand, there are many case where multiple copies of the same good IK solution dominate the ranking (if there are multiple joints and (n-1) match your current configuration closely, 2*PI deviation in only one joint is easily ranked better than smaller deviations on multiple joints).

Since this duplication is relative to the power set of the underconstrained multirotational joints, this can quickly lead to having to scan hundreds of near identical solutions to find the top few actual candidate solutions.  This is actually specifically why I had to implement this check in the first place.